src/themePicker.ts

```diff
@@ -32,6 +32,9 @@ const themes: ThemeItem[] = [
   new ThemeItem("Everglade green", 104),
   new ThemeItem("Buttered toast", 105),
   new ThemeItem("Dark mauve", 200),
+  new ThemeItem("Terminal", 300),
```

Comment:
```suggestion
  new ThemeItem("DarkFlagshipTerrastruct", 201),
  new ThemeItem("Terminal", 300),
```

https://github.com/terrastruct/d2/blob/66ecf134b45b32d4fc2bc37dba3b8043599d6494/d2themes/d2themescatalog/catalog.go#L10-L33